include/deal.II/numerics/vector_tools_integrate_difference.templates.h

```diff
@@ -502,7 +504,7 @@ namespace VectorTools
               update_flags |= UpdateFlags(update_normal_vectors);
             DEAL_II_FALLTHROUGH;
 
-          default:
+          default: // Linfty, L2 and all others
```

Comment:
This comment is fine IMO since it reminds us that we are still treating `Linfty` and `L2` correctly.

Reply:
Maybe I copied from an older dealii version?! So I just delete the the bullet point 'update comments' and drop this commit from the PR.